src/common/types.ts

```diff
@@ -379,3 +434,29 @@ export enum WebSocketServerStatus {
   STOPPED,
   STARTED,
 }
+
+type ParryggTournament = ParryggTournamentPb.AsObject;
+type ParryggEvent = ParryggEventPb.AsObject;
+type ParryggPhase = ParryggPhasePb.AsObject;
+type ParryggSet = ParryggSetPb.AsObject;
+type ParryggBracket = ParryggBracketPb.AsObject & { sets?: Sets };
+type ParryggSetResult = ParryggSetResultPb.AsObject;
+type ParryggSlot = ParryggSlotPb.AsObject;
+type ParryggSeed = ParryggSeedPb.AsObject;
+
+export type {
+  ParryggTournament,
+  ParryggEvent,
+  ParryggPhase,
+  ParryggBracket,
+  ParryggSet,
```

Comment:
don't do this chain import/export thing, just import from `@parry-gg/client` as needed